localstack/services/apigateway/provider.py

```diff
@@ -1083,11 +1081,36 @@ def update_request_validator(
                 f"Validator {request_validator_id} for API Gateway {rest_api_id} not found"
             )
 
-        patched_validator = apply_json_patch_safe(validator, patch_operations)
-        rest_api_container.validators[request_validator_id] = patched_validator
+        for operation in patch_operations:
```

Comment:
nit: what if we instead of `operation` use `patch_operation` and after instead of `operation_performed` we can use `operation`

Reply:
Thanks @calvernaz for the review 🚀🙏🙂